openmicroscopy/openmicroscopy
# Robotframework : Search tests

To Test:
```
./build.py -f components/tests/ui/build.xml web-chrome -DTEST=search.txt
```
```
./build.py -f components/tests/ui/build.xml web-mozilla -DTEST=search.txt
```
And check if all tests pass. 

Note: Please follow set-up instructions given in "openmicroscopy/components/tests/ui/README.md", before running the test. Please install google chrome as well.
